packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -16,6 +16,8 @@ import deepFreeze from 'deep-freeze';
 
 import { FALLBACK_DAG_MAX_NUM_SERVICES } from './index';
 
+const { version } = require('../../package.json');
```

Comment:
It might be misleading/confusing to display the UI project version rather than the backend release version (which implies a certain UI version).

Reply:
+1. We may need to supply release version to the CONFIG here https://github.com/jaegertracing/jaeger-ui/blob/2a552379c92606c530304cbec33ff4f89d832cad/packages/jaeger-ui/public/index.html#L18-L27

This is replaced in server code with a JSON object.

Reply:
alternative solution that does not touch the config: https://github.com/jaegertracing/jaeger/pull/2547

Reply:
Would you like to give it another try? If you sync to jaeger-ui/master, the index.html now gets the following top level function:
```javascript
// Jaeger version data is embedded by the query-service via search/replace.
function getJaegerVersion() {
  const DEFAULT_VERSION = {"gitCommit":"", "gitVersion":"", "buildDate":""};
  const JAEGER_VERSION = {"gitCommit":"0f88343172c0bb16ad5c1805d18e20d59176ea7e","gitVersion":"v1.20.0","buildDate":"2020-10-09T17:42:48Z"};
  return JAEGER_VERSION;
}
```

Considering that there are several data elements here, I would recommend adding them at the bottom of the About menu, e.g.
  * Jaeger v1.20.0
  * Commit 2a55237
  * Build 2020-10-09T17:42:48Z
  * Jaeger UI v1.9.0

Reply:
Made the updates as requested, though this does make an assumption about the correct short hash for git being 7 characters.  While this is the default, my understanding is that's not always the case if there are unambiguous revisions.  

I think the more technically correct way to get the short hash might be to use `git rev-parse --short <SHA>` at the source and inject it instead, though this would involve more changes on the Jaeger query side.

![image](https://user-images.githubusercontent.com/12620443/96372522-33819000-1135-11eb-8ff5-3a13e1028be0.png)

On a side note, in testing with Jaeger, I found that the [default recommended way to run](https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md#running-local-build-with-the-ui) of `make run-all-in-one` does not include the `$(BUILD_INFO)` necessary to inject the version information, so the about menu appears blank when running locally.  Is this desired or not?

Reply:
Looks awesome. One other idea I had is to make versions actually clickable, to that vN.N.N links would take you to the respective release notes page and the commit hash to that commit (the latter is less important than the versions). But this can be done in a separate PR, I would like to merge this first.

Regarding `make run-all-in-one`, sounds like something we could improve in the make target, although when running from a local copy getting the versions is a bit fuzzy.